### PR TITLE
a little cleanup of doc comments

### PR DESCRIPTION
**Pull request by wickedOne**
be aware of [this one](https://github.com/djoos/Symfony2-coding-standard/compare/master...wickedOne:cleanup?expand=1#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R2) as you might not want this change in the first place (requires some action @ packagist)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "escapestudios/symfony2-coding-standard",
+    "name": "djoos/symfony2-coding-standard",
```
Comment (djoos) [inline composer.json]: Would it be possible to leave this out for the time being? I'll need some more thinking around that as it's [not possible to rename packages on Packagist](https://github.com/composer/packagist/issues/47)...

**Comment by djoos** [comment]
Hi @wickedOne,

"Symfony2-coding-standard" got replaced by "Symfony2 coding standard" in your PR, whereas it should have become "Symfony-coding-standard". Any chance of updating your PR?

I've added a note on the package name change; this needs some more thought because of the way Packagist works.

Thanks!

**Comment by wickedOne** [comment]
updated accordingly...

**Comment by djoos** [comment]
Thanks @wickedOne!

**Pull request merged**